### PR TITLE
downloadform - datadirization

### PR DESCRIPTION
**Pull request by pmauduit**
Same as previous ones, Nothing particular. Even easier because no JSP.

TODO:
* [x] Build
* [x] testsuite
* [x] runtime tests with `georchestra.datadir`
* [x] runtime tests without `georchestra.datadir`

### CODE REVIEW

File: downloadform/pom.xml [hunk 1]
```diff
@@ -147,4 +153,135 @@
...
+          <plugin>
+            <groupId>net.sf.debian-maven</groupId>
+            <artifactId>debian-maven-plugin</artifactId>
+            <version>1.0.6</version>
+            <configuration>
+              <packageName>georchestra-downloadform</packageName>
+              <packageDescription>Debian package for the geOrchestra's download form.</packageDescription>
+              <projectOrganization>geOrchestra</projectOrganization>
+              <maintainerName>Pierre Mauduit</maintainerName>
+              <maintainerEmail>pierre.mauduit@gmail.com</maintainerEmail>
```
Comment (fvanderbiest) [inline downloadform/pom.xml]: PSC

File: downloadform/src/deb/resources/etc/georchestra/downloadform/downloadform.properties [hunk 2]
```diff
@@ -0,0 +1,3 @@
+dlJdbcUrl=jdbc:postgresql://localhost:5432/georchestra?user=www-data&password=www-data
+activated=true
```
Comment (fvanderbiest) [inline downloadform/src/deb/resources/etc/georchestra/downloadform/downloadform.properties]: activated = false by default

**Comment by fvanderbiest** [comment]
Please merge after the above remarks have been addressed. Thanks !

File: downloadform/pom.xml (hunk 1 above)
Comment (pmauduit) [inline downloadform/pom.xml]: done

File: downloadform/src/deb/resources/etc/georchestra/downloadform/downloadform.properties (hunk 2 above)
Comment (pmauduit) [inline downloadform/src/deb/resources/etc/georchestra/downloadform/downloadform.properties]: done

**Pull request merged**